gqlauth/jwt/types_.py

```diff
@@ -183,7 +184,7 @@ def authenticate(cls, info: Info, input_: ObtainJSONWebTokenInput) -> "ObtainJSO
                 UserStatus.unarchive(user)
             if status.verified or app_settings.ALLOW_LOGIN_NOT_VERIFIED:
                 # successful login.
-                user.last_login = localtime()
```

Comment:
If you can add a test for this as well, just run this
https://github.com/nrbnlulu/strawberry-django-auth/blob/42b996af8aacfc6e152afeb2bf87e46a99c026ee/tests/test_login.py#L100
with `USE_TZ=False`